apps/stix-shifter/README.md

```diff
@@ -0,0 +1,84 @@
+STIX-Shifter Threat Bus
+=======================
+
+This app bridges the gap between Threat Bus and commercial security tools by
+leveraging
+[STIX-Shifter](https://github.com/opencybersecurityalliance/stix-shifter).
+
+STIX-Shifter is a tool and library to transform STIX patterns into native
+queries for a variety of (mostly) commercial tools, like
+[IBM QRadar](https://www.ibm.com/security/security-intelligence/qradar) or
+[Splunk](https://www.splunk.com/). Since Threat Bus itself primarily focuses on
+open-source tools, this app now provides a simple way to use intelligence from
+Threat Bus with the commercial tools of your choice.
+
+## How It Works
+
+The `stix-shifter-threatbus` app uses ZeroMQ to connect with Threat Bus. To
+connect via ZeroMQ, users must first install and configure the
+[`threatbus-zmq-app`](https://pypi.org/project/threatbus-zmq-app/) plugin on
+their Threat Bus host.
+
+This app functions as middleman between Threat Bus and commercial security
+products. It subscribes to indicator updates from the bus and uses STIX-Shifter
+to actively translate STIX-2 intelligence to native queries for commercial
+tools. The app then executes these queries via STIX-Shifter. [Result processing
+is yet to be implemented.]
```

Comment:
As a reader, I wonder what's happening with the results? Aren't those the retro sightings?

Reply:
Yes, but that is not yet implemented and will be part of another story (ch24866)